whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -362,6 +367,10 @@ private RdapObject mapCommonConformances(final RdapObject rdapResponse) {
         return rdapResponse;
     }
 
+    private void mapRelationConformances(final RdapObject rdapResponse, final String requestUrl) {
```

Comment:
rename to `includeRirSearchConformance` as `RelationConformance` is too generic

Reply:
Done